lib/pbench/test/unit/agent/task/test_results_push.py

```diff
@@ -162,46 +149,105 @@ def test_multiple_meta_args_single_option():
 
     @staticmethod
     @responses.activate
-    def test_multiple_meta_args():
-        """Test normal operation when all arguments and options are specified"""
+    def test_token_omitted():
+        """Test error when the token option is omitted"""
+
+        runner = CliRunner(mix_stderr=False)
+        result = runner.invoke(main, args=[tarball])
+        assert result.exit_code == 2, result.stderr
+        assert "Missing option '--token'" in str(result.stderr)
 
+    @staticmethod
+    @responses.activate
+    def test_token_envar(monkeypatch):
+        """Test normal operation with the token in an environment variable"""
+
+        monkeypatch.setenv("PBENCH_ACCESS_TOKEN", TestResultsPush.TOKN_TEXT)
         TestResultsPush.add_http_mock_response()
         runner = CliRunner(mix_stderr=False)
+        result = runner.invoke(main, args=[tarball])
+        assert result.exit_code == 0, result.stderr
+        assert result.stdout == ""
+
+    @staticmethod
+    @responses.activate
+    def test_access_error(monkeypatch):
+        """Test error in access value"""
+
+        monkeypatch.setenv("PBENCH_ACCESS_TOKEN", TestResultsPush.TOKN_TEXT)
+        runner = CliRunner(mix_stderr=False)
         result = runner.invoke(
             main,
             args=[
-                TestResultsPush.TOKN_SWITCH,
-                TestResultsPush.TOKN_TEXT,
-                TestResultsPush.ACCESS_SWITCH,
-                TestResultsPush.ACCESS_TEXT,
-                TestResultsPush.META_SWITCH,
-                TestResultsPush.META_TEXT_TEST,
-                TestResultsPush.META_SWITCH,
-                TestResultsPush.META_TEXT_FOO,
                 tarball,
+                TestResultsPush.ACCESS_SWITCH,
+                TestResultsPush.ACCESS_WRONG_TEXT,
             ],
         )
-        assert result.exit_code == 0, result.stderr
-        assert result.stdout == ""
+        assert result.exit_code == 2, result.stderr
+        assert "Error: Invalid value for '-a' / '--access': 'public/private'" in str(
+            result.stderr
+        )
 
     @staticmethod
     @responses.activate
     @pytest.mark.parametrize(
         "status_code,message,exit_code",
         (
-            (HTTPStatus.CREATED, None, 0),
-            (HTTPStatus.OK, {"message": "Dup"}, 0),
-            (HTTPStatus.OK, "Dup", 0),
-            (HTTPStatus.NO_CONTENT, {"message": "No content"}, 0),
-            (HTTPStatus.NO_CONTENT, "No content", 0),
+            (
+                HTTPStatus.CREATED,
+                None,
+                0,
+            ),
```

Comment:
Did you add the trailing comma on line 200 (and 205, et al.)?  When I remove them, Black formats all but two of the tuples on single lines, which I think is easier to read (it re-added the ones for the wrapped lines, so perhaps you provoked it in some way?).

Reply:
It's easy to provoke `black` to anger, so it is no fun: I had added an extra field and then took the extra field away but not the comma. I'm agnostic as to what is easier to read: I'd rather have one line and the fields aligned (but black does not like that either). Anyway, with the shortening of the exception message, only one entry is multiline.